src/android/RSA.java

```diff
@@ -36,22 +44,50 @@
 	public static void createKeyPair(Context ctx, String alias) throws Exception {
 		synchronized (LOCK) {
 			Calendar notBefore = Calendar.getInstance();
+			notBefore.add(Calendar.HOUR_OF_DAY, -26);
```

Comment:
Had an issue using the key pair, because the key wasn't valid when i created it due to the date pointing to the future.
Searched the issue and apparently some devices throw this because of some timezone issue. This back dates de the date by little more than a day as resolved in an similar issue for another app
Exceeding 24 hours makes sure no matter the timezone you'll have a valid key when created